pages/project-version/downloads/views/index.jsx

```diff
@@ -11,6 +11,7 @@ const ProjectDownloads = () => {
   const licenceStatus = thisVersionIsGranted ? project.status : 'inactive';
   const title = get(version, 'data.title') || 'Untitled project';
   const isLegacy = project.schemaVersion === 0;
+  const showProtocols = project.status === 'active' && !isLegacy;
```

Comment:
Any particular reason for the active filter?

Reply:
not anymore - I was meant to remove that. Please hold.